frontend-electron/src/components/Plane.tsx

```diff
@@ -28,6 +31,26 @@ const Plane: FC<Props> = ({
         drop: handleDrop,
     })
     const listenerPos = members[userID]?.pos || { x: 50, y: 50 };
+    const moveUserIcon = (e: any) => {
```

Comment:
1. This should be useCallback with `[listenerPos.x, listenerPos.y]` as the memoization dependency list
 2. Don't use `any`. There's a specific type to apply here (I think HTMLKeyboardEvent or similar?)

Reply:
wow smart